lib/pages/add_instance.dart

```diff
@@ -42,7 +42,11 @@ class AddInstancePage extends HookWidget {
       }
     });
 
-    instanceController.addListener(debounce);
+    useEffect(() {
+      instanceController.addListener(debounce);
+
+      return () => instanceController.removeListener(debounce);
```

Comment:
This won't cleanup since debounce is a new object each time. Luckily controllers dispose automatically so you can return null